src/simd/CMakeLists.txt

```diff
@@ -50,7 +55,7 @@ endmacro ()
 
 simd_add_definitions (DIST_CONTAINS_SSE -DENABLE_SSE=1)
 simd_add_definitions (DIST_CONTAINS_AVX -DENABLE_AVX=1)
-simd_add_definitions (DIST_CONTAINS_AVX2 -DENABLE_AVX2=1)
+#simd_add_definitions (DIST_CONTAINS_AVX2 -DENABLE_AVX2=1)
```

Comment:
remove unused line

Reply:
we have a very useful annotation for this line to fix the avx bug, so shound not remove this now 
![image](https://github.com/user-attachments/assets/616b0c6c-2962-49d2-b726-d720629a59e2)